src/pages/patterns/errors.mdx

```diff
@@ -0,0 +1,311 @@
+---
+title: Error handling
+description: >
+  Error messages let users know that a problem has occurred and what to do about it.
+---
+
+Examples of errors include:
+
+- The system has failed to load or perform an action
+- The user has made an incorrect action or input
+
+Error messages are best surfaced using these components:
+
+- Global Alerts
+- Alerts
+- Notifications
+- Validation messages (in Forms)
+
+## Global Alerts
+
+Use the Global Alert to communicate system level errors that interrupt a user’s task.
```

Comment:
```suggestion
Use the Global Alert to communicate system level errors that interrupt a users task.
```
I don't think contraction was intended here.

Reply:
I think the apostrophe needs to remain here. https://forum.wordreference.com/threads/user%E2%80%99s-users-or-users.1474044/#post-7448965

Reply:
Good call @jzempel